packages/client/mutations/connections/getArchivedTasksConn.js

```diff
@@ -1,8 +1,8 @@
 import {ConnectionHandler} from 'relay-runtime'
 
-const getArchivedTasksConn = (viewer, teamId) =>
-  ConnectionHandler.getConnection(viewer, 'TeamArchive_archivedTasks', {
-    teamId
-  })
-
+const getArchivedTasksConn = (viewer, teamId) => {
+  const connectionFilter = {}
+  if (teamId) connectionFilter['teamId'] = teamId
```

Comment:
nit: try to use dot notation when possible.
to make it work with ts try this.
```ts
const connectionFilter = {} as {teamId?: string}
if (teamId) connectionFilter.teamId = teamId
```
the typings will act as documentation